px-k8s-sql-ha/assets/pxstork.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: v1
```

Comment:
This is no longer needed as latest install.portworx.com now already deploys stork.